client/src/cordova/android/OutlineAndroidLib/outline/build.gradle

```diff
@@ -14,8 +14,8 @@ android {
     }
 
     defaultConfig {
-        minSdk 22
-        targetSdk 34
+        minSdk 24
```

Comment:
How will this affect our support matrix? https://support.google.com/outline/answer/15330818?hl=en

Reply:
Good question. I just want to mention that we should drop support for Android 10 and 11. They've both reached end of life over a year ago

Reply:
@jyyi1 do you have any more concerns with this PR?

Would you like to hold off on submitting it until we have a clean release post-https://github.com/Jigsaw-Code/outline-apps/pull/2567 ?